contracts/state/State.sol

```diff
@@ -145,7 +145,9 @@ contract State is Ownable2StepUpgradeable, IState {
         bytes calldata methodParams
     ) public {
         if (methodId == 1) {
-            uint256 calcId = GenesisUtils.calcIdFromEthAddress(getDefaultIdType(), msg.sender);
+            bytes2 idType = GenesisUtils.getIdType(id);
+            require(_stateData.isIdTypeSupported[idType], "id type is not supported");
```

Comment:
You need to check if idType is supported in `transitState` function too.

Also you can put this check here before `if (methodId == 1) {`, because we may add more methods, but the check would be universal for all of them.

Reply:
After moving this check before if (methodId == 1) - 59 tests is failing
Just double check should I fix this test or leave idTypeCheck inside if (methodId == 1)...?

Reply:
@Kolezhniuk that shouldn't happen. Probably tests use identities created with different idType, so you just need to add this idType to supported after the deployment.

Reply:
Fixed tests